ddl/table.go

```diff
@@ -875,6 +876,11 @@ func (w *worker) onSetTableFlashReplica(t *meta.Meta, job *model.Job) (ver int64
 		return ver, errors.Trace(err)
 	}
 
+	// Ban setting replica count for tables in MySQL system database.
+	if tidb_util.IsSysDB(job.SchemaName) {
```

Comment:
Why do you need to ban it here?

Reply:
Double check?

Reply:
In the rolling upgrade cases, there must be some old jobs passed and set into the queue already. Banning it here is a kind of double-check.